packages/core/src/server.js

```diff
@@ -113,7 +113,7 @@ export default function createPercyServer(percy) {
       .readFile(require.resolve('@percy/dom'), 'utf-8')
       .then(content => {
         let wrapper = '(window.PercyAgent = class PercyAgent { snapshot(n, o) { return PercyDOM.serialize(o); } });';
-        log.deprecated('`percy-agent.js` is deprecated, please update to the latest SDK version');
+        log.deprecated('It looks like you’re using @percy/cli with an older SDK. Please upgrade to the latest version to fix this warning.');
```

Comment:
How do we feel about hard coding a docs URL in here? 😬 https://docs.percy.io/docs/migrating-to-percy-cli

Reply:
I really wish the doc could be managed by the repo (`/docs/*.md`) so we can update the link if the doc moves or changes. Probably fine for now for this log, since this endpoint will likely be removed in a future major version anyway